src/pages/boston/speakers.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import { GatsbyImage as Image, getImage } from 'gatsby-plugin-image';
+import { Button } from 'website-components';
+
+import Seo from '../../components/Seo';
+import SpeakerCard from '../../components/SpeakerCard';
+
+const SpeakersPage = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      speakers: allPeople(sort: { name: ASC }, filter: { attending: { in: ["Boston", "Both"] } }) {
+        nodes {
+          slug
+          name
+          position
+          tags
+          github
+          linkedin
+          twitter
+          attending
+          image {
+            childImageSharp {
+              gatsbyImageData(placeholder: NONE)
+            }
+          }
+        }
+      }
+      heroImage: file(relativePath: { eq: "photos/speakers-barcelona.png" }) {
+        childImageSharp {
+          gatsbyImageData(placeholder: NONE)
+        }
+      }
+    }
+  `);
+
+  return (
+    <>
+      <Seo title="Nextflow Summit 2023 Speakers" />
+      <div className="pt-20 bg-gray-900 text-white">
+        <div className="container-lg">
+          <div className="row lg:flex-nowrap">
+            <div className="col-full lg:col-6">
+              <h1 className="typo-h2 mb-4">More than 25 speakers to look forward to</h1>
+              <p className="typo-body max-w-xl mb-6">
+                The call for speakers and posters will remain open until September 9 or until all places are filled.
```

Comment:
If we are extending the abstract deadline here, we need to extend it throughout the site. Also, September the 9th? Really?